python/graphscope/deploy/kubernetes/resource_builder.py

```diff
@@ -1135,14 +1135,21 @@ def build(self):
 
         return pods_builders, svc_builders
 
-    def build_liveness_probe(self):
-        liveness_cmd = [
+    def build_probe_cmd(self):
+        return [
             "/bin/sh",
             "-ec",
             "ETCDCTL_API=3 etcdctl --endpoints=http://[127.0.0.1]:%s get foo"
             % str(self._listen_client_service_port),
         ]
-        return ExecProbeBuilder(liveness_cmd, timeout=15, failure_thresh=8)
+
+    def build_liveness_probe(self):
+        liveness_cmd = self.build_probe_cmd()
+        return ExecProbeBuilder(liveness_cmd, timeout=15, period=10, failure_thresh=8)
+
+    def build_readiness_probe(self):
```

Comment:
You can try to check `self._listen_peer_service_port` by `httpGet` in `readiness probe`